.tx/config

```diff
@@ -19,6 +19,24 @@ source_file = docs/build/gettext/usage.pot
 source_lang = en
 type = PO
 
+[multigeiger.docs_build_gettext_assemblyguide]
+file_filter = locales/<lang>/LC_MESSAGES/assemblyguide.po
+source_file = docs/build/gettext/assemblyguide.pot
+source_lang = de
+type = PO
```

Comment:
We talked about this: this is temporarily using the direction de -> en, until we have an english translation

Then it should be reversed to have the usual en -> de (or in general: en -> others) direction.

Reply:
> Hmm, github says "empty file"?

This is just a representation issue in github, because these files cannot be compared automatically. If you click on the three dots on the right hand side and click view, you can download the file from this pull request.

To make it easier for you, here is the deep link: https://github.com/ecocurious2/MultiGeiger/tree/2e890f07b15216f914f34069bd61980366cd82db/docs/Bauanleitung

Do you have further questions? If yes, I could join your meeting today at 20:30 to discuss final questions and to say good bye to the team.